editor/components/inserter/menu.js

```diff
@@ -131,15 +131,15 @@ export class InserterMenu extends Component {
 				return recentItems;
 
 			case 'blocks':
-				predicate = ( item ) => item.category !== 'embed' && item.category !== 'reusable-blocks';
```

Comment:
There's a few more to update:

https://github.com/WordPress/gutenberg/search?utf8=%E2%9C%93&q=reusable-blocks&type=